language/tools/vm-genesis/src/lib.rs

```diff
@@ -93,6 +94,9 @@ pub fn encode_genesis_change_set(
     create_and_initialize_validators(&mut genesis_context, &validators);
     reconfigure(&mut genesis_context);
 
+    // XXX/TODO: for testnet only
```

Comment:
My main question about this PR was how we are going to implement this. Let's sit down with @davidiw next week 
 and talk about how to manage the common and distinct genesis blobs for all the deployments?

Reply:
It's a good question :) The genesis additions are separable, so I'll land this now to avoid rebase hell, and then once we have a good separation strategy I'll change it.